player-tester/src/comparator-handler.js

```diff
@@ -0,0 +1,26 @@
+import ChartManager from "./chart";
+
+const chartContainerElt = document.getElementById("chart-containers");
+
+document.getElementById("import").onclick = function() {
+    const files = document.getElementById("file").files;
+    console.log(files)
+    if (files.length <= 0) {
+        window.alert("No file selected")
+        return false;
+    }
+    const fileReader = new FileReader();
+    fileReader.onload = (event) => {
+        const data = JSON.parse(event.target.result)
```

Comment:
Just a question, I didn't test this , but if we import two files, our chart will always refer to the first file ( as we don't remove the files from the document files, i.e `document.getElementById("file").files` will always have 2 files, and `event.target.result` will always give the same file descriptor no ?